src/main/flManager/application_business_rules/flManRequestModel.java

```diff
@@ -0,0 +1,59 @@
+package flManager.application_business_rules;
+
+import java.util.List;
+
+public class flManRequestModel {
+    private String owner;
+    private List<String> friends;
+    private int value;
+
+    public flManRequestModel(String owner){
+        this.owner = owner;
+    }
+
+    public String getOwner() {
+        return owner;
+    }
+
+    public void setOwner(String owner) {
+        this.owner = owner;
+    }
+
+    public List<String> getFriends() {
+        return friends;
+    }
+
+    public void setFriends(List<String> friends) {
+        this.friends = friends;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public void setValue(int value) {
+        this.value = value;
+    }
+
+    public void addFri(String user){
+        this.friends.add(user);
+    }
+
+    public void delFri(String user){
+        this.friends.remove(user);
+    }
+
+    public static String viewFri(String owner){
```

Comment:
I'm not sure where you would use this so you might be able to get rid of it. However, if you have a thought about where you need it, feel free to leave it